styles/nitrotype/catppuccin.user.css

```
@@ -0,0 +1,1421 @@
+
+@var checkbox styleRaceBox "Style Race Box" 1
+@var checkbox preserveGoldGradient "Keep the gold lower gradients in boxes." 0
+==/UserStyle== */
+
+@-moz-document domain('nitrotype.com') {
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    body {
+      background: url("/dist/site/images/backgrounds/bg-tires.png"), @base;
+      border-color: @border !important;
+      color: @text !important;
+    }
+
+    /* NT-specific vars */
+    /* color */
+    /* the border surrounding a mantle box */
+    @border: @overlay0;
+    @text-shadow: @border;
+    @tooltip: @overlay0;
+    @input-field: @surface0;
+    @colored-button-text: @crust;
+    @tab-selected: @overlay0;
+    @row1: @surface2;
+    @row2: @surface1;
+
+    /* Shared Components */
+    /* link */
+    a {
+      color: @blue;
+    }
+    /* colors generic */
+    .tc-emerald,
+    .link--emerald {
+      color: @green;
+    }
+    /* password required */
+    .input-require,
+    .input-optional {
+      color: @red;
+    }
+    /* progress bar */
+    .prog--c .prog-barFill {
+      background-color: @blue;
+    }
+    /* gradient banner */
+    .bg--gradient,
+    .bg--gradient2 {
+      background: @crust;
+    }
+    /* table */
+    .table {
+      border-bottom-color: rgba(@crust, 0.1);
```

Surprised this even works lol.
```suggestion
      border-bottom-color: fade(@crust, 10%);
```